src/deviceApiCalls/transports/extension.transport.js

```diff
@@ -55,7 +55,6 @@ async function extensionSpecificRuntimeConfiguration (deviceApi) {
     const contentScope = await getContentScopeConfig()
     const emailProtectionEnabled = isAutofillEnabledFromProcessedConfig(contentScope)
     const incontextSignupEnabled = isIncontextSignupEnabledFromProcessedConfig(contentScope)
-    const incontextSignupDismissedAt = await deviceApi.send(new GetIncontextSignupDismissedAtCall(null))
```

Comment:
As mentioned above, this appears to be user-state related and not actually relevant to 'global' settings, so it's moved into the feature file